packages/app-store/routing-forms/pages/router/[...appPages].tsx

```diff
@@ -34,14 +37,15 @@ const querySchema = z
     slug: z.string(),
     pages: z.array(z.string()),
   })
-  .catchall(z.string());
+  .catchall(z.string().or(z.array(z.string())));
```

Comment:
Support comma separated string as well as multiple params with same name for multiple select.